### PR TITLE
Health check responses consolidation + adding request ID to responses

### PR DESCRIPTION
**Pull request by jwair**
Consolidated health check response objects
Added missing request ID to the response

### CODE REVIEW

**Comment by wilkermichael** [comment]
@mkam AJ and I were talking about this PR, so I can go ahead and review it

File: api/health_test.go [hunk 1]
```diff
@@ -30,11 +30,9 @@ func Test_HealthHandler_GetHealth(t *testing.T) {
 			validate: func(t *testing.T, resp *httptest.ResponseRecorder) {
```
Comment (wilkermichael) [inline api/health_test.go:30]: I think we should also add checks that we are setting the requestID as expected

You can define some request ID outside the for loop and then just add it to the context of the request
```Go
path := "/v1/health"
req, err := http.NewRequest(http.MethodGet, path, nil)
require.NoError(t, err)
resp := httptest.NewRecorder()

req = req.WithContext(requestIDWithContext(req.Context(), reqID))
```

**Review by wilkermichael — APPROVED** [review]
One comment on testing, otherwise looks good!

Thanks for doing this!

File: api/health_test.go (hunk 1 above)
Comment (jwair) [reply api/health_test.go:30]: Thanks, I'll add that

Comment (jwair) [reply api/health_test.go]: Addressed in https://github.com/hashicorp/consul-terraform-sync/pull/834/commits/9aea60d770d2b39bcc18b4495bd173cb5e46dfe1

**Review by wilkermichael — APPROVED** [review]

**Pull request merged**